README.md

```diff
@@ -19,10 +45,210 @@ This library is built atop of [PSR-7](https://www.php-fig.org/psr/psr-7/) and
 composer require partnero/partnero-php
 ```
 
+<a name="usage"></a>
+# Usage
+
+<a name="partners-api"></a>
+## Partners for Affiliate program
+
+<a name="get-a-list-of-partners"></a>
+### Get a list of partners
+
+```php
+use Partnero\Partnero;
+
+$partnero = new Partnero("api_key");
+
+$partnero->partners()->list(limit: 10);
+```
+
+<a name="get-partner"></a>
+### Get partner
+
+```php
+use Partnero\Partnero;
+
+$partnero = new Partnero("api_key");
+
+$partnero->partners()->find(key: 'partner-key');
+```
+
+<a name="create-partner"></a>
+### Create partner
+
+```php
+use Partnero\Partnero;
+use Partnero\Models\Partner;
+
+$partnero = new Partnero("api_key");
+
+$partner = (new Partner())
+  ->setEmail('test@mail.com')
+  ->setName("Name")
+  ->setSurname("Surname")
+  ->setKey("partner-key");
+
+$partnero->partners()->create($partner);
+```
+
+Surname and Key are optional.  
+If key is not set, a random key will be generated for the partner.
+
+<a name="update-partner"></a>
+### Update partner
+
+```php
+use Partnero\Partnero;
+use Partnero\Models\Partner;
+
+$partnero = new Partnero("api_key");
+
+$partner = (new Partner())
+  ->setEmail('john.doe@mail.com')
```

Comment:
Maybe we need to be consistent with double or single quotes. I'm voting for single.